govuk_admin_template.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "jquery-rails", "~> 4.3.1"
   gem.add_dependency "plek", ">= 2.1.0"
   gem.add_dependency "rails", ">= 3.2.0"
```

Comment:
Should we be setting this to a higher version? As I understand by having a Ruby 2.7.2 constraint Rails 6 is about the earliest version that will be supported.

Reply:
👍🏻 updated to rails >= 6